firebase-perf/src/test/java/com/google/firebase/perf/metrics/TraceTest.java

```diff
@@ -80,7 +81,9 @@ public void setUp() {
     ConfigResolver.clearInstance();
 
     context.getSharedPreferences(PREFS_NAME, Context.MODE_PRIVATE).edit().clear().commit();
-    ConfigResolver.getInstance().setApplicationContext(context);
+    ConfigResolver configResolver = ConfigResolver.getInstance();
+    configResolver.setDeviceCacheManager(new DeviceCacheManager(new FakeDirectExecutorService()));
+    configResolver.setApplicationContext(context);
```

Comment:
I think we can remove this. Is any test erroring out?

Reply:
Similar with the above comment on (setIsPerformanceCollectionEnabled).